synapseutils/copy.py

```diff
@@ -398,7 +398,7 @@ def _getSubWikiHeaders(wikiHeaders, subPageId, mapping=None):
     """
     Function to assist in getting wiki headers of subwikipages
     """
-    subPageId = str(subPageId)
+    subPageId = str(int(subPageId)) if isinstance(subPageId, float) else str(subPageId)
```

Comment:
This commit is not related to submitting docker entity, please put it in a separate pull request. Otherwise, please include full test coverage for this change.

Reply:
Oh! I didn't see this!  Sorry about that.  I removed it